### PR TITLE
Add update notifier

### LINKED ISSUE #65: Add update notifier
Is this something people like and would want?  We would use https://github.com/yeoman/update-notifier like yeoman does.

cc @sindresorhus

### PR DESCRIPTION
**Pull request by tdeschryver**
Closes #65

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -47,6 +47,7 @@
     "pretty-hrtime": "^1.0.0",
     "semver-greatest-satisfied-range": "^1.0.0",
     "tildify": "^1.0.0",
+    "update-notifier": "^1.0.3",
```
Comment (tdeschryver) [inline package.json]: This is the latest version supporting node 0.10.

**Comment by phated** [comment]
Thanks @tdeschryver - we're going to hold off on this a little bit because it adds a lot of overhead to startup time and I want to use libraries that actually support node 0.10 at latest.

**Comment by tdeschryver** [comment]
@phated sure, no problem. 
I was a bit hesitant for the same reason you mentioned.
I'll take a look at the open issues in the next week if that's OK :smile:

**Comment by phated** [comment]
I'm going to close this for the reasons above.  We can revisit the issue down the road.

**Pull request closed without merging**